linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml

```diff
@@ -25,11 +25,13 @@ references:
     stigid@rhel6: RHEL-06-000213
     srg@rhel6: SRG-OS-000095
     nist@rhel6: CM-7
+    nist-csf@rhel6: PR.IP-1,PR.PT-3
     disa@rhel6: '305'
     disa: "381"
     hipaa: 164.308(a)(4)(i),164.308(b)(1),164.308(b)(3),164.310(b),164.312(e)(1),164.312(e)(2)(ii)
     iso27001-2013: A.8.2.3,A.13.1.1,A.13.2.1,A.13.2.3,A.14.1.2,A.14.1.3
     nist: AC-17(8),CM-7(a)
+    nist-csf: PR.AC-3,PR.IP-1,PR.PT-3,PR.PT-4
```

Comment:
Another rhel6 vs generic ``nist-csf``. Should the second one be nist-csf@rhel7?

Reply:
@shawndwells no because as it stands right now the generic `nist-csf` is essentially saying all products except `rhel6` map to this.